core/editable.js

```diff
@@ -1717,14 +1718,24 @@
 			// Split inline elements so HTML will be inserted with its own styles.
 			path = range.startPath();
 			if ( ( node = path.contains( isInline, false, 1 ) ) ) {
-				range.splitElement( node );
+				clone = range.splitElement( node );
```

Comment:
A bit misleading thus this element is not a clone but the late element after the split. Maybe rename it into `endNode`?